src/wymeditor/plugins/table/jquery.wymeditor.table.js

```diff
@@ -138,22 +138,38 @@ TableEditor.prototype.bindEvents = function () {
     jQuery(wym._box).find(
         tableEditor._options.sAddRowButtonSelector
     ).click(function () {
-        return tableEditor.addRow(wym.selectedContainer());
+        var cont = wym.selectedContainer();
+        if (cont) {
+            tableEditor.addRow(cont);
+        }
+        return false;
     });
     jQuery(wym._box).find(
         tableEditor._options.sRemoveRowButtonSelector
     ).click(function () {
-        return tableEditor.removeRow(wym.selectedContainer());
+        var cont = wym.selectedContainer();
```

Comment:
What does `selectedContainer()` return if the collection is across multiple blocks? I think at least some of the table methods handle a multiple selection, so we need to make sure we're not breaking that.